### PR TITLE
[Php80][CodingStyle] Handle ClassPropertyAssignToConstructorPromotionRector+NewlineAfterStatementRector

### PR DESCRIPTION
**Pull request by samsonasik**
Handle stuck on used of 2 rules:

```php
Rector\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector;
Rector\CodingStyle\Rector\Stmt\NewlineAfterStatementRector;
```

Closes https://github.com/rectorphp/rector-src/pull/1346 Fixes https://github.com/rectorphp/rector/issues/6840

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php [hunk 1]
```diff
@@ -151,6 +151,10 @@ public function refactor(Node $node): ?Node
             }
         }
 
+        if ($this->nodesToRemoveCollector->isActive()) {
```
Comment (Bl00D4NGEL) [inline rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php]: Couldn't this be moved to the top of the refactor function to reduce a little bit of runtime?

Comment (samsonasik) [reply rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php]: moved to top https://github.com/rectorphp/rector-src/pull/1350/commits/e2056ec8a4d324aa457a558788e6613d755dfeb0

**Comment by TomasVotruba** [comment]
Thank you :+1: 

@samsonasik Feel free to merge when ready.

**Pull request merged**